tests/functional/test_download.py

```diff
@@ -61,6 +70,81 @@ def test_download_wheel(script, data):
     assert script.site_packages / 'piptestpackage' not in result.files_created
 
 
+def _get_abi_tag():
```

Comment:
super nit: defining this inside the test, since it's a one-test ~pony~ helper.

Reply:
Originally the legacy PyPy test also used it, but then I realized that case could be simplified by forcing fake ABI and platform tags.